src/tests/ftest/cart/util/cart_logtest.py

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python3
 #
-# Copyright 2018-2023 Intel Corporation
+# (C) Copyright 2018-2023 Intel Corporation
```

Comment:
Any specific reason for adding the "(C)"?  It has no legal value in copyright law.

Reply:
The commit hook wasn't triggering for some reason so I changed it to be consistent with other files but I'm not sure if it actually helped.  Will revert this line on next push.